code/__DEFINES/misc.dm

```diff
@@ -49,7 +50,7 @@
 #define R_HAND_LAYER			3		//Having the two hands seperate seems rather silly, merge them together? It'll allow for code to be reused on mobs with arbitarily many hands
 #define BODY_FRONT_LAYER		2
 #define FIRE_LAYER				1		//If you're on fire
-#define TOTAL_LAYERS			26		//KEEP THIS UP-TO-DATE OR SHIT WILL BREAK ;_;
+#define TOTAL_LAYERS			27		//KEEP THIS UP-TO-DATE OR SHIT WILL BREAK ;_;
 
```

Comment:
Having two layers for mutations is required for two reasons: 
- We want to be able to have both hulk and laser eyes shown at the same time. 
- We want hulk to be behind body layer (to fix the underwear bug) but laser eyes must appear above the mob's eyes.